packages/bddeb

```diff
@@ -99,6 +101,36 @@ def write_debian_folder(root, templ_data, cloud_util_deps):
                              params={'build_depends': ','.join(requires)})
 
 
+def write_debian_folder_from_branch(root, templ_data, branch):
+    """Import a debian package directory from a branch."""
+    print("Importing debian/ from branch %s to %r" % (branch, root))
+
+    p_dumpdeb = subprocess.Popen(
+        ["git", "archive", branch, "debian"], stdout=subprocess.PIPE
+    )
+    subprocess.check_call(
+        ["tar", "-v", "-C", root, "-x"],
+        stdin=p_dumpdeb.stdout
+    )
+
+    print("Adding new entry to debian/changelog")
+    full_deb_version = (
+        templ_data["version_long"] + "-1~bddeb" + templ_data["release_suffix"]
```

Comment:
Did we close on whether this value ("-1~bddeb") needs to be user-controllable?

Reply:
Like this is behaves like plain `bddeb` without `--packaging-branch`. We can certainly make it configurable, but is it useful in practice?